Automattic/jetpack
# Connection: hoist Identity Crisis package back into Connection

## Proposed changes:
* Move the Identity Crisis package contents into Connection package.
* Abandon the Identity Crisis package.
* Ensure backward and forward compatibility for existing code.

### Other information:


## Jetpack product discussion
Related to p9dueE-4Ep-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Create two JN sites (Site A and Site B), switch them to the branch. Connect both, then disconnect Site B.
2. Use Broken Token to copy the tokens and blog ID from Site A to Site B.
3. Confirm that Site B displays IDC screen. Check on Jetpack and non-Jetpack pages.
4. Try "staying in safe mode", confirm that the IDC screen disappears and the toolbar button appears. Click on the "Safe Mode" toolbar button, confirm IDC screen shows up again.
5. Choose to create new connection, confirm connection gets established.
6. Disconnect Site B again, copy tokens and blog ID from Site A.
7. Confirm you see the IDC screen. Resolve it by migrating the data.
8. Confirm the connection got migrated, and cache site's URL's are updated.
